src/main/java/it/gov/pagopa/payhub/activities/activity/utility/util/Constants.java

```diff
@@ -0,0 +1,13 @@
+package it.gov.pagopa.payhub.activities.activity.utility.util;
+
+import java.time.format.DateTimeFormatter;
+
+public class Constants {
+    public static final String MAIL_TEXT = "mailText";
+    public static final String ACTUAL_DATE = "actualDate";
+    public static final String FILE_NAME = "fileName";
+    public static final String TOTAL_ROWS_NUMBER = "totalRowsNumber";
+    public static final String REPORTING_PATH = "RENDICONTAZIONE";
+    public static final DateTimeFormatter MAIL_DATE_FORMAT =  DateTimeFormatter.ofPattern("EEE, MMM dd yyyy, hh:mm:ss");
+    public static final String WS_USER = "-WS_USER";
```

Comment:
these constants doesn't have any purpose than the class that is using them, let's move them there, removing this class

Reply:
Tha class has not been removed, but removed onluy service constants.  Pachage was also changed